src/Oceananigans.jl

```diff
@@ -48,7 +48,7 @@ export
     # Turbulence closures
     IsotropicDiffusivity, AnisotropicDiffusivity,
     AnisotropicBiharmonicDiffusivity,
-    ConstantSmagorinsky, AnisotropicMinimumDissipation,
+    SmagorinskyLilly, ConstantSmagorinsky, AnisotropicMinimumDissipation,

```

Comment:
Let's pick one to export! How about `SmagorinskyLilly` only?

Reply:
Then we can delete the alias which is a very positive thing.

Reply:
I honestly don't know when/why `ConstantSmagorinsky` came about. I kept it there because it was already there, but I definitely prefer `SmagorinskyLilly`. If we're not gonna export it, do we need that alias?

Reply:
Although an argument could be made that it's only a Smagorinsky-Lilly model if `C_s=0.16`, since that was mostly Lilly's contribution... but I think that's okay